changelog.txt

```diff
@@ -7,6 +7,14 @@ The **signac** package follows `semantic versioning <https://semver.org/>`_.
 Version 2
 =========
 
+[2.4.0] -- 2025-xx-xx
+---------------------
+
+Fixed
++++++
+
+ - Schema detection when a state point was a list of dictionaries having lists (#1030).
```

Comment:
A state point must be a dict at the root. Do you mean “a dictionary of lists containing dictionaries (possibly of lists)?”